inlong-sdk/dataproxy-sdk/src/main/java/org/apache/inlong/sdk/dataproxy/config/ProxyConfigManager.java

```diff
@@ -632,7 +632,7 @@ private ProxyConfigEntry getProxyConfigEntry(DataProxyNodeResponse proxyCluster)
         if (ObjectUtils.isNotEmpty(proxyCluster.getIsSwitch())) {
             isSwitch = proxyCluster.getIsSwitch();
         }
-
+        int maxPacketLength = nodeList.get(0).getMaxPacketLength() != null ? nodeList.get(0).getMaxPacketLength() : 0;
```

Comment:
It is better to use a negative number when not set. 0 is also an allowed length.